docs/source/documentation.rst

```diff
@@ -324,9 +324,9 @@ To segment the same image by using the **'pixel_size_in_micrometer.txt'** file i
 Segment multiple images of the same resolution
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To segment multiple microscopy images of the same resolution that are located in the same folder, specify the path to the folder in the **-i** argument. For instance, to segment the images in folder **'test_sem_image/image 1_sem/'** of the test dataset that have a pixel size of 0.07 micrometers, use the following command::
```

Comment:
There are some`_v1` left in the models in doc on lines 289-290. We could use this PR to correct the names.

Reply:
@mariehbourget my laptop battery just died 😅, if you can please pull my branch, make the change, and push it. Thanks!

Reply:
(or make the change through the github interface)